backend/api/Controllers/MissionSchedulingController.cs

```diff
@@ -179,6 +179,23 @@ ISourceService sourceService
                 missionRun.CalculateEstimatedDuration();
             }
 
+            IList<MissionStatus> missionStatuses = [MissionStatus.Ongoing, MissionStatus.Pending, MissionStatus.Paused];
+            var returnToHomeMission = await missionRunService.ReadReturnToHomeMissionRuns(robot.Id, missionStatuses);
```

Comment:
This function feels both quite specific and trying to be general. Could we make it even more specific or more general? For example "ReadUnfinishedReturnToHomeMissionsRuns" or "ReadMissionRuns" and mission type is an input?

Reply:
I personally like the latter.

Reply:
Have adressed this